### DISCUSSION TITLE
Is there a way to manually add a function start address and run recursive disassembly?

### DISCUSSION THREAD

**Discussion by Cryspia** [radareorg/radare2#20781] [Q&A]
I tried to use `af+ FUNC_OFFSET`, but the function seems to be of 0 size in the function table. The `aaaa` and `pdr` command refuses to analysis or disassemble from the function I inputted and tells me `Cannot find function in FUNC_OFFSET`. I wonder what is the correct way to manually add a function for disassembly if I only know the function start offset.

**Comment by trufae** [comment]
afr

**Comment by trufae** [comment]
If you want to manuay create all the basic blocks associated with a function entry that you ate creating see af+ and afb+ actually if you do a2f or af* is what you get and whats saved in projects to reconstruct the analysis.

the function is 0 in size because it containe 0 basic blocks
